src/magic_folder/magic_file.py

```diff
@@ -194,6 +214,9 @@ class MagicFile(object):
     _queue_remote = attr.ib(default=attr.Factory(list))
     _is_working = attr.ib(default=None)
 
+    # XXX
```

Comment:
What is this "XXX" supposed to signify? Should this be expanded into a comment? If not, remove.